static/app/components/charts/eventsGeoRequest.tsx

```diff
@@ -90,7 +105,16 @@ const EventsGeoRequest = ({
       // Prevent setState leaking on unmounted component
       mounted = false;
     };
-  }, [query, yAxis, start, end, period, environments, projects, api]);
+  }, [
+    api,
+    eventView,
+    organization.slug,
+    referrer,
+    prevApi,
+    prevEventView,
+    prevOrgSlug,
+    prevReferrer,
```

Comment:
iirc, refs aren't valid as a dependency because mutating them won't trigger an update

Reply:
it's the result of usePrevious, not a ref anymore because that hook unpacks it: 

https://github.com/getsentry/sentry/blob/c022e914ff546f6eb86ce4b6f2224f490875fb4c/static/app/utils/usePrevious.tsx#L20

Also, eslint asks for it:
<img width="557" alt="Screen Shot 2022-11-17 at 1 32 26 PM" src="https://user-images.githubusercontent.com/187460/202564299-8c49b180-ea73-4bb7-8074-bfddf3e8e195.png">